src/pages/profile/index.js

```diff
@@ -0,0 +1,53 @@
+import "./profile.scss";
+
+import {profileData} from "../../data/profileData"
+
+import {activateChangingAvatarModal} from "./modules/changeAvatarModalWindow/changeAvatarModalWindow.js";
+import {showProfileData, changeProfileData, showChangePasswordSection} from "./profile.js"
+
+document.addEventListener('DOMContentLoaded', () => {
+    let avatar = document.getElementById("avatarImg");
+    avatar.src = profileData.imgSrc;
+    
+    let userName = document.querySelector(".userName");
+    userName.textContent = profileData.display_name;
```

Comment:
getElementById, querySelector могут вернуть null. Правильней использовать if, либо [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)
```suggestion
    let avatar = document.getElementById("avatarImg");
    avatar?.src = profileData.imgSrc;
    
    let userName = document.querySelector(".userName");
    userName?.textContent = profileData.display_name;
```
Если значение не будет меняться то по конвенции используют const а не let